Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -11695,20 +11447,6 @@ public struct PrimaryAssociatedTypeClause: SyntaxBuildable, ExpressibleAsPrimary
     assert(rightAngleBracket.text == ">")
   }
 
-  /// A convenience initializer that allows:
-  ///  - Initializing syntax collections using result builders
-  ///  - Initializing tokens without default text using strings
-  public init(
-    leftAngleBracket: TokenSyntax = TokenSyntax.`leftAngle`,
-    rightAngleBracket: TokenSyntax = TokenSyntax.`rightAngle`,
-    @PrimaryAssociatedTypeListBuilder primaryAssociatedTypeListBuilder: () -> ExpressibleAsPrimaryAssociatedTypeList = { PrimaryAssociatedTypeList([]) }
```

Comment:
Please keep this as `PrimaryAssociatedType` conforms to `HasTrailingComma`

Reply:
❌ Again, IMO the dominating use case is that there is a single primary associated type so there’s no need for a result builder.